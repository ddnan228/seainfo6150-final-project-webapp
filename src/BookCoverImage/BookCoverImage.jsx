import React from 'react'
import css from "./BookCoverImage.module.css";

const BookCoverImage = ({url, text}) => {
  return (
    <div className = {css.bookCoverStyle}>
      <img src={url} alt={text} />
    </div>
  );
};

export default BookCoverImage;
