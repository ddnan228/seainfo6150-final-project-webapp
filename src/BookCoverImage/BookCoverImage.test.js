import React from "react";
import { render } from "@testing-library/react";
import BookCoverImage from "./BookCoverImage.jsx";

it("renders the BookCoverImage component correctly", () => {
  const url="https://images-na.ssl-images-amazon.com/images/I/9195S4FTnOL.jpg";

  const { container } = render(<BookCoverImage url={url} />);
  expect(container).toMatchSnapshot();
});
