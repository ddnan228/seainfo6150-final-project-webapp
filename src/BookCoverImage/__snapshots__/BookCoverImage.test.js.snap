// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the BookCoverImage component correctly 1`] = `
<div>
  <div
    class="bookCoverStyle"
  >
    <img
      src="https://images-na.ssl-images-amazon.com/images/I/9195S4FTnOL.jpg"
    />
  </div>
</div>
`;
